benchmarks/graphics_pipeline/main.cpp

```diff
@@ -615,26 +661,28 @@ void ProjApp::Render()
             // Draw sphere instances
             uint32_t pipeline_index = pKnobVs->GetIndex() * kAvailablePsShaders.size() + pKnobPs->GetIndex();
             frame.cmd->BindGraphicsPipeline(mPipelines[pipeline_index]);
-            frame.cmd->BindIndexBuffer(mSphere.mesh);
```

Comment:
Is there a reason that you want to avoid using index buffer?

Reply:
No, there isn't. In fact, the number of indices is the same as the number of vertices, so I could also use indices instead of vertices. I preferred to use vertices instead of indices to ensure that we don't need to worry if we are reusing indices. Here we are not reusing indices.

Reply:
I thought we could reuse vertices within the sphere instance?

Reply:
Fixed. I have updated the code so that we now reuse vertex data using their indices. For example, for a sphere with a longitude = 10 and a latitude = 10, we are using (10+1)*(10+1) = 121 vertices, but 600 indices to draw the sphere. This allows us to efficiently reuse vertex data through their corresponding indices.